packages/api-contract/src/Abi/toLatest.spec.ts

```diff
@@ -65,6 +65,16 @@ describe('v1ToLatest', (): void => {
     ).toEqual(['flip', 'get']);
   });
 
+  it('has the correct messages with namespaced method name', (): void => {
+    const contract = registry.createType('ContractMetadata', { V1: abis.ink_v1_psp22.V1 });
+    const latest = v1ToLatest(registry, contract.asV1);
+    expect(
+      latest.spec.messages.map(({ label }) => label.toString())
+    ).toEqual(['PSP22Metadata,token_name', 'PSP22Metadata,token_symbol', 'PSP22Metadata,token_decimals', 'PSP22Mintable,mint',
```

Comment:
```suggestion
    ).toEqual(['PSP22Metadata::token_name', 'PSP22Metadata::token_symbol', 'PSP22Metadata::token_decimals', 'PSP22Mintable::mint',
```

I believe the above is how it is now in master, let's see what the CI tests throw out.